packages/paste-website/src/pages/components/page-header/index.mdx

```diff
@@ -86,15 +88,15 @@ There are 3 sections that make up the Page Header component, though you won’t
 
 1. **PageHeaderSetting** contains any components that relate to the positioning of the current page in relation to parent or sibling pages, including navigating between those pages. Context components may include:
     - [Breadcrumb](/components/breadcrumb)
-    - [Anchor](/components/anchor) (e.g. “<-- Back to parent page” link)
+    - [Anchor](/components/anchor) (for example, “<-- Back to parent page” link)
     - [Progress Steps](/components/progress-steps)
 2. **PageHeaderDetails** is where all page-related information belongs. It includes static text and interactive components that relate to the page as a whole. Components that might live within the PageHeaderDetails include:
-    - [Detail Text](/components/detail-text) (e.g. the title of the entire wizard flow)
-    - [Heading](/components/heading) (e.g. the page title)
-    - [Badge](/components/badge) (e.g. “enabled”, “beta”, etc.)
-    - [Paragraph](/components/paragraph) (e.g. additional text below the title)
-    - [Button](/components/button) and [Button Group](/components/button-group) (e.g. “create”, “edit”, “cancel”, etc. buttons)
-    - [Anchor](/components/anchor) (e.g. link to a related/useful page such as docs - not a parent page)
+    - [Detail Text](/components/detail-text) (for example, the title of the entire wizard flow)
+    - [Heading](/components/heading) (for example, the page title)
+    - [Badge](/components/badge) (for example, “enabled”, “beta”, etc.)
```

Comment:
```suggestion
    - [Badge](/components/badge) (for example, “beta”, etc.)
```

chore: Remove "enabled" since we give guidance in Composition notes to only use it for release stages.